LiteFarmOrg/LiteFarm
# LF-2528 sensor map preview compact temperature component

Ticket [LF-2528](https://lite-farm.atlassian.net/browse/LF-2528)

**This PR also resolves the issue from #2132  where sensor cluster didn't have long press**

To Test:
1. Ideal for #2132 to be merged first before reviewing this PR.
2. Have sensor temperature reading data (could be dummy data in the database as long as they're mapped correctly to a sensor in your farm map). The creation date must be within the last 24 hrs (the map preview will only show the most recent 24hr data)
3. Long press a sensor location on the map to see the preview of the sensor data.
4. Try long press a sensor from sensor cluster on the map to see the preview of the sensor data.
5. For the same sensor reading type (i.e. temperature), you should only see the most recent reading.
